apps/src/javalab/JavalabView.jsx

```diff
@@ -94,6 +104,15 @@ class JavalabView extends React.Component {
     return <div id="visualization" />;
   };
 
+  setHeaderHeight = () => {
+    let editorAndVisualization = document.getElementById('editor-and-vis-div');
```

Comment:
we're mixing react and the DOM API paradigms here -- use a react ref instead, which will have similar properties (e.g., being able to get the element's top boundary), but will make sure everything plays nicely with react's shadow DOM

Reply:
Gotcha- updated. Thanks!